indrasetiawan.is-not-a.dev.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "check this out",
+
+  "domain": "is-not-a.dev",
+  "subdomain": "indrasetiawan",
+
+  "owner": {
+    "repo": "https://github.com/Gand0r/Gand0r.github.io",
+    "email": "portgash.the.ace@gmail.com"
+  },
+
+  "record": {
+    "CNAME": "https://gand0r.github.io"
```

Comment:
Cannot have https or / in records.

Reply:
like this ? 

"CNAME": "gand0r.github.io"